packages/@react-native-windows/cli/src/runWindows/utils/msbuildtools.ts

```diff
@@ -87,7 +87,13 @@ export default class MSBuildTools {
       `/flp2:warningsonly;logfile=${warnLog}`,
     ];
 
-    if (!singleproc) {
+    // Building projects in parallel increases compiler memory usage and
+    // doesn't lead to dramatic performance gains (See #4739). Only enable
+    // parallel builds on machines with >16GB of memory to avoid OOM errors
+    const highMemory = totalmem() > 16 * 1024 * 1024 * 1024;
```

Comment:
please make this >=

Reply:
That wouldn't change anything in practice, since the numbers reported by totalmem aren't perfectly round. E.g. on my machine with 32GB of RAM:
```
> os.totalmem() / 1024 / 1024 / 1024
31.93259048461914
```

Guessing there is a portion reserved by the system (The issue isn't 1024 vs 1000).

The logic here in effect means we get parallel builds enabled on machines with specs listed that are greater than 16GB of RAM. That number is a little arbitrary, but Kiki had repeated issues building on a laptop with 16GB of memory, and I've seen issues with that number before, so I moved the bar a bit higher.